org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/SignatureHelpHandler.java

```diff
@@ -278,7 +276,7 @@ private static boolean isSameParameters(IMethod method1, IMethod method2, Comple
 		}
 		String[] params1 = method1.getParameterTypes();
 		String[] params2 = method2.getParameterTypes();
-		if (params2.length == params1.length - 1) {
+		if (params2.length <= params1.length - 1) {

```

Comment:
I guess this makes sense because any time a candidate signature has more arguments, but matches on the existing ones typed out so far, it might still be the active. Is there an example that this fixes ?

Reply:
See
- https://github.com/eclipse/eclipse.jdt.ls/blob/master/org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/SignatureHelpHandlerTest.java#L267
-

Reply:
I think `testSignatureHelp_parameterTypes` is the example for this one and `testSignatureHelp_constructorParameters3` for the one above, but they both show the need for the changes.